packages/Articles/resources/views/partials/category.array.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+$this->extract($category, [
+    'id',
+    'name',
+    'alias',
+    'image',
+    'description',
+    'order',
+    'status',
+]);
+$this->set('parentId', $category->parent_id);
+$this->set('createdAt', date('c', strtotime($category->created_at)));
```

Comment:
Think to a helper for this date format? We can easier change format or even check if date is invalid.